docs/framing.md

```diff
@@ -2,6 +2,46 @@
 
 Framing wraps messages with headers and checksums so receivers can identify message boundaries and verify integrity.
 
+## Frame Format Definitions
+
+All supported frame formats are defined in [`examples/frame_formats.proto`](../examples/frame_formats.proto). This file provides:
+
+- Protocol Buffer definitions for each frame format
+- Enumeration of all frame format types
+- Configuration message for runtime format selection
+
+## Recommended Frame Formats
+
+| Format | Preamble | Header | CRC | Length | Total Overhead | Use Case |
+|--------|----------|--------|-----|--------|----------------|----------|
+| NoFormat | 0 | 0 | 0 | None | 0 | Trusted links, nested protocols |
+| BasicFrame | 1 | 2 | 2 | None | 4 | Standard reliable communication |
```

Comment:
All Recommended Frame Formats have to include MSG ID. Remove the No Format Frame.